### DISCUSSION TITLE
Model YYS-2518 is supported?

### DISCUSSION THREAD

**Discussion by rmoggia** [roleoroleo/yi-hack-MStar#412] [Q&A]
Hello everyone.
The model YYS-2518 I do not find it among those supported (at least I think). Did I search well or did I make some mistakes?

**Comment by roleoroleo** [comment]
I need a backup of the firmware to check if it's supported.

**Reply by rmoggia** [reply]
Thanks for the reply. So I back up the firmware, install the custom firmware and test it.
If that doesn't work, restore the original firmware. Did I understand well? (I'm sorry but I'm not an expert). A thousand thanks.

**Reply by roleoroleo** [reply]
No, backup the firmware and send me the zip.
I will read internally the model and I will check it it's compatibile with the hack.
